Integrations/integration-OpsGenie.yml

```diff
@@ -0,0 +1,141 @@
+description: Get current on-call assignments and users info
+detaileddescription: OpsGenie integration allows querying for the current on-call
+  users for a given Schedule. The integration requires an API key, that can
+  be acquired from the OpsGenie interface, under Integrations / API.
+configuration:
+- display: API Key
+  name: apiKey
+  defaultvalue: ""
+  type: 4
+  required: true
+- display: User system proxy configuration
+  name: proxy
+  defaultvalue: "true"
+  type: 8
+  required: false
+- display: Base URL
+  name: baseURL
+  defaultvalue: https://api.opsgenie.com/v2
+  type: 0
+  required: true
+script:
+  script: |
+    var baseURL = params.baseURL;
+    if (baseURL[baseURL.length - 1] != '/') {
+        baseURL += '/';
+    }
+
+    var serviceURLs = {
+        schedules: 'schedules',
+        account: 'account',
+        users: 'users'
+    };
+
+    var sendRequest = function(url, param) {
```

Comment:
you can remove params here as its not passed inside the call but a global

Reply:
This is `param` and not `params`. it's a way to pass parameters to the API. 
But I'm not using it with the current APIs implemented, so removed,.